Dockerfile

```diff
@@ -1,26 +1,27 @@
 # Build the manager binary
-# Build the manager binary
-FROM golang:1.16 as builder
+FROM --platform=${BUILDPLATFORM} golang:1.16 as builder
```

Comment:
hadolint complains about this https://github.com/hadolint/hadolint/wiki/DL3029
I guess due to that it will make it harder to run a normal docker build command.
Would it be possible to use the solution they suggest?

Reply:
Ah, looks like that’s a false positive from hadolint. `${BUILDPLATFORM}` is set by BuildKit itself to be the platform you’re running the build on.

Reply:
(The reason I’m doing this here is Go can cross-compile easily by just setting `GOARCH`. Doing it the way hadolint suggests will require compiling the arm64 operator under qemu. CI runs much faster this way, only using qemu for the last steps to build the final image.)

Reply:
Okay, learning every day, thanks for all the good background information.

Can you please add `.hadolint.yaml` to this PR with the following content:

```.hadolint.yaml 
ignored:
  - DL3029
```

and we will ignore the error that holds back the CI right now.